knowledge-base/kb-radwindow-prismapplication-createshell.md

```diff
@@ -0,0 +1,52 @@
+---
+title: CreateShell() method can't return RadWindow in Prism 7
+description: 
```

Comment:
Please add description and page_title. Check the KB Content Standard document to see how the title and description should look.
[https://wiki.progress.com/pages/viewpage.action?spaceKey=DTS&title=Knowledge+Base+Content+Standards](https://wiki.progress.com/pages/viewpage.action?spaceKey=DTS&title=Knowledge+Base+Content+Standards)